docs/guidelines/platforms/android/guideline_percievable_android.md

```diff
@@ -164,41 +221,28 @@ Grouping elements based on the context helps users that benefit from using acces
 </ConstraintLayout>
 ```
 
-_Note:_ In cases like this, it is recommended to define content descriptions as concisely as possible, considering that accessibility services will read them one after the other.
-
-- Custom group label
-
-It is possible to override the platform's default grouping and ordering of a group's inner element descriptions by providing a content description for the group itself.
-
-**Code example:**
-
+In Compose, the same can be achieved using the `mergeDescendants` parameter:
+```kotlin
+@Composable
+private fun SongCard(title: String, artist: String) {
+    Column(modifier = Modifier.semantics(mergeDescendants = true) {}) {
+        Text(title)
+        Text(artist)
+    }
+}
 ```
-<!-- In response to a single user interaction, accessibility services announce the custom content description for the group. -->
-<ConstraintLayout
-android:id="@+id/song_data_container" ...
-android:screenReaderFocusable="true"
-android:contentDescription="@string/title_artist_best_song">
 
-    <TextView
-        android:id="@+id/song_title" ...
+_Note 1:_ In cases like these, it is recommended to define content descriptions as concisely as possible, considering that accessibility services will read them one after the other.
 
-        <!-- Content ignored by accessibility services -->
-        android:text="@string/my_song_title" />
-    <TextView
-        android:id="@+id/song_artist"
-
-        <!-- Content ignored by accessibility services -->
-        android:text="@string/my_songwriter" />
-</ConstraintLayout>
-```
+_Note 2:_ It is also possible to define a `contentDescription` for the whole group, in which case the descriptions of the children will be ignored. But setting the description for the whole group by aggregating the descriptions of its descendants is not advised, as it is error-prone. For example, if any of the descendants' text changes, the group description will not be updated automatically.
```

Comment:
I would add the link from your [comment](https://github.com/infinum/accessibility-mobile-standards/pull/59/files#r1822996612) here because it is more transparent on why this is not a preferred practice

Reply:
Sounds good! 👍